webapp/app/js/perf.js

```diff
@@ -268,7 +338,8 @@ perf.controller('PerfCtrl', [ '$state', '$stateParams', '$scope', '$rootScope',
                                   { project: series.projectName,
                                     signature: series.signature,
                                     visible: series.visible})); }),
-                                'highlightedRevision': $scope.highlightedRevision
+                                'highlightedRevision': $scope.highlightedRevision,
+                                'zoom': JSON.stringify($scope.zoom)
```

Comment:
While you're here, could you make the indentation the same as for the object elements above?